doc/contributing.md

```diff
@@ -0,0 +1,125 @@
+
+Search existing issues
+Be sure to search existing open and closed issues before filing a new issue: Nobody enjoys reading 20 reports of the same issue - we'd MUCH prefer it if you add comments to an existing issue describing the same/similar issue.
+
+This project is moving fast and its likely someone else has found the problem you're seeing, and someone may be working on or have already contributed a fix!
+
+If not, great - please file a new issue:
+
+### File a new Issue
+
+When you hit "New Issue", select the type of issue closest to what you want to report/ask/request:
+!(/docs/images/new-issue-template.png)
+
+**Complete the requested information and provide as much information as possible**. The more information you provide, the more likely your issue/ask will be understood and fixed. Wherever possible, provide steps to reproduce the issue: **Repro steps are among the most valuable part of a bug report!**
+
+Other Helpful information includes:
+
+* What device you're running (inc. CPU type, memory, disk, etc.)
+* What build of Windows your device is running
+  
+  👉 Tip: run `ver` in PowerShell/Cmd, e.g.:
+
+    ```powershell
+    C:\> ver
+
+    Microsoft Windows [Version 10.0.18900.1001]
+    ```
+
+* What tools and apps you're using (e.g. VS 2019, VSCode, etc.)
+* Don't assume we're experts in setting up YOUR environment and don't assume we are experts in `<your distro/tool of choice>`. Teach us to help you!
+* What steps do we need to take to reproduce the issue? Assume we love to read repro steps. As much detail as you can stand is probably _barely_ enough detail for us!
+* If you're reporting a particular character/glyph not rendering correctly, the specific Unicode codepoint would be MOST welcome
+* Prefer error message text where possible, or screenshots of errors if text cannot be captured
+
+👉Tip: We MUCH prefer text command-line script than screenshots of command-line script.
+
+## Proposing a new feature or a fix to an issue
+
+⚠ Again, to help streamline your proposed fix or new feature, **file an issue and discuss your issue/feature/ask with the team before starting any work.**
+
+⚠ Also, remember that **changes you make may affect both Windows Terminal and Windows Console and may end up being re-incorporated into Windows itself!** Because of this, we will treat community PR's with the same level of scrutiny and rigor as commits submitted to the official Windows source by team members and partners.
+
+### To Spec or not to Spec
+
+Some issues/features may be quick and simple to describe and agree upon. For such scenarios, once a team member has agreed your approach, skip ahead to "Fork, Branch, and Create your PR". Small issues that do not require a spec will be labelled Issue-Bug.
+

```

Comment:
nit: awkward wording - missing the word with 

> For such scenarios, once a team member has agreed your approach

Reply:
@bitcrazed this one still exists. "has agreed your approach"

Reply:
Fixed